package.json

```diff
@@ -10,9 +10,11 @@
   },
   "main": "index.js",
   "scripts": {
+    "build": "npm run clean:css",
     "example:fastatic": "examples/fastatic/index.js",
     "commit": "git-cz",
     "test": "tape test/*.js | tap-spec",
+    "clean:css": "cleancss -o src/assets/voorhoede-core.min.css src/assets/voorhoede-core.css",
```

Comment:
I don't think we need an advanced CSS build step, since the CSS isn't going to change regularly.
Doesn't the minified code need to be in the `assets` directory since it's used by the  templates?